project_index.php

```diff
@@ -0,0 +1,27 @@
+<?php
+require_once('functions.php');
+$db = new PDO('mysql:host=db; dbname=collection_project', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT `charname`, `class`, `level`, `strength`, `dexterity`, `constitution`, `intelligence`, `wisdom`, `charisma` FROM `characters`;");
+// $query->bindParam(':email', $email);
+$query->execute();
+$characters = $query->fetchAll();
+
+echo displayCharacters($characters);
+// echo $characters[0]['charname'];
+echo '<pre>';
+var_dump($characters);
+echo '</pre>';
+
+
+?>
+
+<html>
+    <head>
+    </head>
+    <body>
+        <div>
+            <!-- <p>Name: <?php echo $characters[1]['charname'];?></p> -->
```

Comment:
You can strip this line out entirely and replace it with what you currently have on line 10 

`echo displayCharacters($characters);` 

This way the character list will be output in the right place